docs/cluster_scaling_schedules_crd.yaml

```diff
@@ -12,6 +12,10 @@ spec:
     listKind: ClusterScalingScheduleList
     plural: clusterscalingschedules
     singular: clusterscalingschedule
+    shortNames:
+      - css
+      - clustersched
+      - clusterschedule
```

Comment:
These files are generated, the change needs to be done in https://github.com/zalando-incubator/kube-metrics-adapter/blob/master/pkg/apis/zalando.org/v1/types.go (There is some annotation to generate this)

Reply:
Oh you're right, let me take another look at this.

Reply:
Updated. It looks like the `all` category was previously missing from the `cluster_scaling_schedules` as there were 2 `kubebuilder:resource` annotations defined with only the final one actually taking effect (`scope`).